src/app/routes/homePage/getInitialData/page-config.ts

```diff
@@ -0,0 +1,10 @@
+type Envs = 'test' | 'live';
```

Comment:
Should we rename this file to `home-page-config.ts` to match the exported `HOME_PAGE_CONFIG`?